docs/nidigital/class.rst

```diff
@@ -4007,7 +4007,7 @@ instrument_manufacturer
 
     .. py:attribute:: instrument_manufacturer
 
-        Returns a string ("National Instruments") that contains the name of the manufacturer of the digital pattern instrument.
+        Returns a string ("NI") that contains the name of the manufacturer of the digital pattern instrument.
```

Comment:
We didn't update the driver to return `NI` as part of brand identity updates; it still returns `National Instruments`.